src/index.js

```diff
@@ -39,7 +39,7 @@ RootControlComponent.prototype.rootUpdate = function(_control) {
 extend(RootControlComponent, BaseComponent);
 
 function createNode({ el, nodeProps, component }) {
-  const comp = component.component || new NodeComponent(nodeProps);
+  const comp = Object.create(component.component.prototype, nodeProps) || new NodeComponent(nodeProps);
```

Comment:
But what if is component.component invalid?

Reply:
Outdated. Please address in proper commit.